portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uisections/CKEditor.path

```diff
@@ -95,6 +95,11 @@
 	<td></td>
 </tr>
 <!--TOOLBAR-->
+<tr>
```

Comment:
Hi @LucasAcacio ,

Although the CKEditor.path file exists this test is in the context of forms, we prefer that Paths be kept in our own files. Could you please remove this path from this file and add it on the FormFields.path file instead.

Thanks!